zerver/openapi/zulip.yaml

```diff
@@ -3770,6 +3770,12 @@ paths:
                                 Unlike the simpler [realm/update](#realm-update) event format, this
                                 event type supports multiple properties being changed in a
                                 single event.
+
+                                **Changes**: The `email_address_visibility` setting was removed in Zulip 7.0 (feature level 162).
+                                It was replaced by a [user setting](/api/update-settings) with a
+                                [realm user default](/api/update-realm-user-settings-defaults), with the encoding of different
+                                values preserved. Clients can support all versions by supporting the current API and treating
+                                every user as having the realm's email address visibility value.
```

Comment:
I guess the fact that the encoding is the same as it was before makes this a lot easier to write -- that perhaps argues for not standardizing to the `common_policy_types` encoding with this migration.